src/coreclr/ToolBox/superpmi/superpmi-shared/compileresult.h

```diff
@@ -211,5 +211,7 @@ class CompileResult
     MemoryTracker*          memoryTracker;
     Capture_AllocMemDetails allocMemDets;
     allocGCInfoDetails      allocGCInfoDets;
+
+    const bool recordCallSitesWithoutSig = false; // Set it to true if you want to use CallUtils::GetRecordedCallSiteInfo.
```

Comment:
who sets this to `true`?

Reply:
you can set it to true (and rebuild) if you want to debug SuperPMI manually and planning to call `CallUtils::GetRecordedCallSiteInfo`. There are no callers that read these values nowadays, so another option was to delete these dead functions but maybe somebody is using them from VS/WinDBG.

Reply:
got it.